common/core/src/retry_operation.ts

```diff
@@ -32,11 +32,11 @@ export class RetryOperation {
    * @param {(opCallback: (err?: Error, result?: any) => void) => void} operation The operation to execute.
    * @param {(err?: Error, result?: any) => void} finalCallback                   The callback to call with the final error or result, after retries if necessary.
    */
-  retry(operation: (opCallback: (err?: Error, result?: any) => void) => void, finalCallback: (err?: Error, result?: any) => void): void {
+  retry(operation: (opCallback: (err?: Error, result?: any, response?: any) => void) => void, finalCallback: (err?: Error, result?: any, response?: any) => void): void {
     const retryOperation = () => {
       this._retryCount++;
       /*Codes_SRS_NODE_COMMON_RETRY_OPERATION_16_001: [The `operation` function should be called at every retry.]*/
-      operation((err, result) => {
+      operation((err, result, response) => {
```

Comment:
>result [](start = 22, length = 6)

aargh.  this again.  For DPS, i've been using "result" for the protocol-specific object and "body" or "responseBody" for the body of the response.  I've also been putting body before result because it's more useful for consumers.  (result is more for diagnosing).  You were right earlier.  We do need to standardize on this and I'm hoping it's not too bad.